dockerfiles/Dockerfile_slim

```diff
@@ -9,7 +9,7 @@ RUN mkdir /usr/local/lib/nodejs \
   && tar xf nodejs.tar.gz -C /usr/local/lib/nodejs/ --strip-components 1 \
   && rm nodejs.tar.gz node-file-lock.sha
 RUN apt-get install --assume-yes \
-  openjdk-11-jdk-headless=11.0.7+10-2ubuntu2~18.04 \
+  openjdk-11-jdk-headless \
```

Comment:
So we don't have any java version limitations?

Reply:
I thought the only limitation we have is that it's jdk 11 but not a minor or patch version of 11.

Reply:
You can see the different jdk version [here](https://www.oracle.com/java/technologies/javase-downloads.html)

Reply:
Limitation is on java major version. But, @jonnyhork have you tested if the flavor matters. "2ubuntu2~18.04"?

Reply:
I'm not sure how to test that actually, the build runs and all the dependencies install like sfdx. What else is dependent on Java? I don't understand what the "flavor" is and why it will matter _after_ a successful install.

From [this site](https://www.digitalocean.com/community/tutorials/how-to-install-java-with-apt-on-ubuntu-18-04)
> By default, Ubuntu 18.04 includes OpenJDK version 11, which is an open-source variant of the JRE and JDK.

It sounds like jdk comes with ubuntu 18.04, I figured the version downloaded by apt-get will be compatible.

Reply:
Apex language server is the only one that is dependent on Java. From reading the link it sounds like this should work. let's give it a try and see.

Reply:
@vazexqi you wrote this: https://github.com/forcedotcom/salesforcedx-docker/commit/4355ccc88e9422e231a6cad4edea84248c64451b I see that you switched to a specific version of Java. Any reason?